juniper_official/Subscriber-Service/address-pool-utilization.rule

```diff
@@ -0,0 +1,292 @@
+iceberg {
+    topic service.subscribers {
+        description "Rules related to subscribers address pool utilization";
+        synopsis "BNG subscribers address pool monitoring rules";
+        rule address-pool-utilization {
```

Comment:
Please change the rule names to start with a verb. Ex: check-address-pool-utilization.

Reply:
Not able to add check because of length limit.

Reply:
Can you please change this to chk-addr-pool-util or some such.